sos/plugins/ovn_central.py

```diff
@@ -25,29 +25,47 @@ class OVNCentral(Plugin):
     plugin_name = "ovn_central"
     profiles = ('network', 'virt')
     option_list = [
-        ('log_dir', 'Directory where OVN logs are found', '',
-         '/var/log/openvswitch/'),
-        ('db_dir', 'Directory where OVN databases are found', '',
-         os.environ.get('OVS_DBDIR', '/var/lib/openvswitch/')),
-        ('runtime_dir', 'Directory where OVN runtime files are found', '',
-         '/var/run/openvswitch/'),
+        ('log_dir', 'Directory where OVN logs are found', '', ''),
+        ('db_dir', 'Directory where OVN databases are found', '', ''),
+        ('runtime_dir', 'Directory where OVN runtime files are found', '', ''),
         ('schema_dir', 'Directory where OVN database schema files are found',
          '', '/usr/share/openvswitch/'),
     ]
 
+    def get_dir(self, plugin_var, *candidates):
+        # If the user specified a plugin option, definitely
+        # use that value.
+        our_dir = self.get_option(plugin_var)
+        if our_dir:
+            return our_dir
+
+        for candidate in candidates:
+            if candidate and os.path.exists(candidate):
+                return candidate
+
```

Comment:
Missing return value if out_dir = '' and no candidate matches - then None is returned, what raises exception in `os.path.join(None,...)`